lib/builder/builder.go

```diff
@@ -128,6 +130,7 @@ func (c *Config) CheckAndSetDefaults() error {
 		c.FieldLogger = logrus.WithField(trace.Component, "builder")
 	}
 	if c.Progress == nil {
+		// FIXME: 6
```

Comment:
Currently, progress reporting output is broken w.r.t to number of build steps which I'm no longer sure how many atm (and I should not care really - this needs fixing).